scrypto-compiler/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "scrypto-compiler"
+version = "1.2.0-dev"
+edition = "2021"
+
+[dependencies]
+radix-engine-interface = { path = "../radix-engine-interface", default-features = false }
+radix-engine = { path = "../radix-engine", default-features = false }
```

Comment:
I think it is not so straight forward because `extract_definition` internally uses some modules from `radix-engine` to execute wasm file. For instant it uses: `WasmModule`, `WasmRuntime`, `SystemLoanFeeReserve`.